### PR TITLE
Ldelossa/phase out sec label pt1

### PR DESCRIPTION
**Pull request by ldelossa**
Currently in the datapath we swap around the terminology `sec_label` and `identity`. 

As an effort to consolidate our terminology, this PR begins the phasing out of the keywords `sec_label` in favor of `sec_identity`.

An arbitrary starting point was picked, the `remote_endpoint_info` struct, and further renames will take place over time, as to not create too many conflicts with other PRs. 

```release-note
Rename the `sec_label` field in remote_endpoint_info structure to `sec_identity`
```

### CODE REVIEW

File: bpf/bpf_host.c [hunk 1]
```diff
@@ -101,18 +101,18 @@ ipcache_lookup_srcid6(struct __ctx_buff *ctx)
 	struct remote_endpoint_info *info = NULL;
```
Comment (aspsk) [inline bpf/bpf_host.c]: `also update all update variables` -- too many `update`s

File: bpf/lib/host_firewall.h [hunk 2]
```diff
@@ -16,417 +16,417 @@
 
 # ifdef ENABLE_IPV6
 static __always_inline int
-ipv6_host_policy_egress(struct __ctx_buff *ctx, __u32 src_id,
```
Comment (aspsk) [inline bpf/lib/host_firewall.h:19]: This diff is huge because tabs were replaced by spaces (in other places too). Please, push a new version

**Review by aspsk — CHANGES_REQUESTED** [review]

File: bpf/lib/host_firewall.h (hunk 2 above)
Comment (ldelossa) [reply bpf/lib/host_firewall.h:19]: Ugh yeah, wrong editor profile. Will fix.

File: bpf/bpf_host.c (hunk 1 above)
Comment (ldelossa) [reply bpf/bpf_host.c]: Fixed in newest commit

**Review by aspsk — APPROVED** [review]
LGTM from technical perspective.

I am not a big fan of this `_sec_` thing though. Do we have other types of identities? If we want to emphasize that this is a "Secutiry Identity" then I would rather use `security_id` than `sec_identity`.

**Comment by ldelossa** [comment]
We landed on "sec_identity" since it stays close to the terminology used in our upstream documentation. 

I.e. we make explicit call outs to the term "identity" in docs. For instance you can search "identity" in upstream docs and get a lot of info. 

I understand this is a bit subjective, but there was some previous discussion on this naming change, and I think we could keep going back and forth on 'security_id' vs 'sec_identity' ad infinitum. My feeling is to try to ground the naming change into something tangible, like what words we use in the upstream docs the most often to explain a "concept".

**Comment by ldelossa** [comment]
/test-1.25-5.4

**Pull request merged**